packages/atlas/src/components/AppKV/AppKV.styles.ts

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled'
+
+import { Text } from '@/components/Text'
+import { media, sizes } from '@/styles'
+
+export const Wrapper = styled.div`
+  position: relative;
+`
+
+export const Image = styled.img`
+  object-fit: contain;
+  width: 100%;
+`
+
+export const StyledText = styled(Text)`
+  display: flex;
+`
+
+export const LogoWrapper = styled.div`
+  position: absolute;
+  inset: 0;
+  display: flex;
+  gap: ${sizes(6)};
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+export const Fade = styled.div`
+  position: absolute;
+  inset: 0;
+  top: 40%;
+  background: linear-gradient(180deg, rgb(24 28 32 / 0) 0%, #181c20 100%);
```

Comment:
we should use her `cVar(colorBackground)` for `#181c20` and we could replace `rgb(24 28 32 / 0)` with `transparent`(I think)